stock_by_warehouse_sale/views/sale_view.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-    <record id="sale_order_form_view" model="ir.ui.view">
+    <record id="view_order_form" model="ir.ui.view">
```

Comment:
Be careful changing `xmlid` since that it will need a migration script but there is not

Reply:
changing xml_id  of views without noupdate=1 will drop those views, and create them again, isn't it?

Reply:
I'm not sure

Could you confirm that, please?

Reply:
I'm indeed sure. Records not marked as noupdate may be safely modified, including their external IDs.

As @hugho-ad mentioned, if they're renamed, old ones will be automatically removed and new ones created.